SQL/2015-06-11-subprojectConfigTable.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE subproject (
```

Comment:
we have a proj_rel table that contains list of subprojects for every project, does it make sense to just add ProjectID column to this table?